plugins/statistics.koplugin/main.lua

```diff
@@ -353,6 +376,8 @@ function ReaderStatistics:getDatesFromAll(sdays, ptype)
     local one_day = 24 * 3600 -- one day in seconds
     local avg_time_per_page
     local period = now_stamp - ((sdays -1) * one_day) - from_begin_day
+    local week_translate = _("Week")
```

Comment:
Do you need to declare a variable here if it's never changed? Why not just use `_("Week")` directly inside the if block.